src/Confluent.Kafka/Impl/LibRdKafka.cs

```diff
@@ -966,57 +966,48 @@ internal static ErrorCode committed(IntPtr rk, IntPtr partitions, IntPtr timeout
         private static Func<IntPtr, IntPtr, ErrorCode> _position;
         internal static ErrorCode position(IntPtr rk, IntPtr partitions)
             => _position(rk, partitions);
+        
```

Comment:
nit: there's some unnecessary white space here and other places (check with git diff).

Reply:
you only fixed one superfluous whitespace edit. there are more. good to fix, though this won't stop me merging.